map2.js

```diff
@@ -31,3 +31,18 @@ var phoneBook = [
 // For example, the 0th element of the new array should be {Abe: "1-111-111-1111"}
 
 //Your code here
+const getNum = phoneBook.map(function(item) {
+  const strName = Object.keys(item).toString()
+  //converts key to string
+  const num = Object.values(item).toString()
+  //converts value to string
+  const firstNum = num.charAt()
+  //grab character at index 0
+  const concNum = firstNum+'-'+num
+  //create a new phone number that appends prefix captured in firstNum
+  return {[strName]: concNum}
+  //save object that has key of person's name and value of updated phone number to the new array
+})
```

Comment:
great work with this! your comments really help elucidate what you were thinking, and your reasoning is 💯 . 

I ran the code here and it's interesting to me that you put `strName` in square brackets in the return statement. I expected it to turn the key into an array, but it didn't; learned something new today 😄 . That said, the square brackets in line 43 aren't necessary. Out of curiosity, what was your reasoning for putting them in?

Reply:
I can't remember why I added square brackets - maybe my mind was on arrays? I'll go in and update that. Thanks for the feedback!